services/deployment.md

```diff
@@ -25,6 +25,13 @@ $ pulbot deploy my_app/my_branch to staging
 
 You can also deploy to other environments (like `production`) or deploy branches with CI failures by using `deploy!`.
 
+### Deploying a new app
+
+You must add heaven's public key to the deploy user's authorized_keys on the box that will be deployed to.
+
+`curl https://raw.githubusercontent.com/pulibrary/princeton_ansible/master/keys/heaven.pub >> cat authorized_keys`
```

Comment:
I think you need to get rid of the `cat` here.

Reply:
haha, oops, thanks